ui/app/helpers/stringify-object.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Changes a JSON object into a string
+ */
+
+import { helper } from '@ember/component/helper';
+
+export default helper(function stringifyObject(positional /*, named*/) {
+  return JSON.stringify(positional[0], null, 2);
```

Comment:
`question`:  Should this get some tests around it to explain the whitespace argument?
`nit`: Should we opt for named arguments? Or refactor `positional` to be something like:

```
function stringifyObject([obj]) {
  return JSON.stringify(obj, null, 2)
}
```

Alternatively, we could write this API

```
function stringifyObject([obj], { replacer = null, whiteSpace = 2 }) {
  return JSON.stringify(obj, replacer, whiteSpace);
}
```